pkg/helpers/helpers.go

```diff
@@ -188,3 +189,28 @@ func GetBrokerTokenAndCA(client kubernetes.Interface, brokerNS, clusterName stri
 	return "", "", fmt.Errorf("ServiceAccount %v/%v does not have a secret of type token", brokerNS, clusterName)
 
 }
+
+var (
+	clusterInfoGVR schema.GroupVersionResource = schema.GroupVersionResource{
+		Group:    "internal.open-cluster-management.io",
+		Version:  "v1beta1",
+		Resource: "managedclusterinfos",
+	}
+)
+
+func GetClusterType(dynamicClient dynamic.Interface, clusterName string) (string, error) {
+	clusterInfo, err := dynamicClient.Resource(clusterInfoGVR).Namespace(clusterName).Get(context.TODO(), clusterName, metav1.GetOptions{})
```

Comment:
you can get that from vendor label in managedCluster. do you?

Reply:
yes, I can get vendor label in managedcluster, but the label may be customized by users.

Reply:
I think we should reduce the dependence on other apis, do we need consider if there is no ACM installed? can we provide a flag (or an env) for submariner-addon, we may try to get the cluster type by `managedclusterinfos` api firstly, if it fails, we try to get the cluster type from the flag? or keep this simple, we get the cluster type from vendor label and doc this

Reply:
i would go with label only. We might have some api changes on managedcluster later  to include more informations, but label is good enough is the  first step.

Reply:
done